.github/workflows/jira-issue-create.yml

```diff
@@ -0,0 +1,34 @@
+name: Jira Issue Creator
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    environment: Jira
+    name: SDK Bot Jira Issue Creation
+    steps:
+      - name: Login
+        uses: atlassian/gajira-login@master
+        env:
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+          JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+      - name: Create issue
+        id: create
+        uses: atlassian/gajira-create@master
+        with:
+          project: ${{ secrets.JIRA_PROJECT }}
+          issuetype: Task
+          summary: |
+            [SDK - Node] ${{ github.event.issue.title }}
+          description: |
+            ${{ github.event.issue.url}}
```

Comment:
would love to be more prescriptive with this description - e.g. reminders to respond to the creator (so that it doesn't seem like we're ghosting them while we working on it), reminders to close out the issue.

Reply:
Would this be some chron job that looks through past issues?